CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [kirkstone-0.24.0] Q4 2023
+- bootloader-update (breaking change for current release):
+  - introduction of bootloader update for phytec tauril2 and raspberry pi4
+  - introduction of bootloader update for welotec eg500
```

Comment:
why not a single line or three subpoints?

Reply:
done